portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/StructuredContentFilteringAndSorting.testcase

```diff
@@ -77,15 +77,15 @@ definition {
 	test CanFilterEqualToPriorityValuesSortedByPriorityAscendingByDefault {
 		property portal.acceptance = "true";
 
-		task ("When web contents are filtered with 1.0 priority and sorted with default priority") {
+		task ("When web contents are filtered with greater than 1.0 priority and sorted with default priority") {
 			var response = HeadlessWebcontentAPI.filterAndSortAdminStructuredContent(
-				filtervalue = "priority%20eq%201.0",
+				filtervalue = "priority%20gt%201.0",
```

Comment:
I changed it because with priority 1.0 it would only return 1 WC and to have more than one answer I put it greater than 1.0 and adjusted the answer to the expected one, for this test and for others.